LocalWiki.php

```diff
@@ -424,6 +424,10 @@ function onBeforeInitialize( Title &$title, $unused, OutputPage $output, User $u
 	case 'sagan4alphawiki':
 		$wgCargoAllowedSQLFunctions[] = 'RAND';
 		break;
+	case 'simpleelectronicswiki': 
```

Comment:
Could add these settings to ManageWiki.

Reply:
https://github.com/miraheze/mw-config/pull/5309